_data/glossary.yaml

```diff
@@ -81,11 +77,6 @@
   develop, ship, and run applications
   - The docker daemon process running on the host which manages images and containers
   (also called Docker Engine)
-<a class="glossary" name="docker_enterprise">Docker Enterprise</a>: |
```

Comment:
Should we instead keep this and again re-cycle the content Kory created for the header of the enterprise pages as the definition here?

Reply:
Hi Ben, I'm not sure what content that is. @KoryKessel-Docker Can you provide more information? Thanks!

Reply:
@traci-morrison Ben is referring to the Enterprise note Kory is working on. For now, you can leave the Enterprise section 'as is'. When we add the note on the Enterprise pages, we can add it here too. Hope that makes sense.

Reply:
@traci-morrison, I agree with @usha-mandya in that we should coordinate putting the acquisition blurb here once it is fully approved and starts going up elsewhere. As of now, it reads as follows:

"The Docker Enterprise platform has been acquired by Mirantis, inc., effective 13-November-2019. For more information on the acquisition and how it may affect you and your business, refer to the [Docker Enterprise Customer FAQ](https://www.docker.com/faq-for-docker-enterprise-customers-and-partners)."